src/blender/scripts/colors.py

```diff
@@ -252,8 +250,8 @@ def load_font(font_path):
 except:
 	bpy.context.scene.render.image_settings.file_format = params["file_format"]
 	bpy.context.scene.render.image_settings.color_mode = params["color_mode"]
```

Comment:
The `try:` part of all these blocks:

```py
bpy.context.scene.render.file_format = params["file_format"]
bpy.context.scene.render.color_mode = params["color_mode"]
```
should almost definitely just be eliminated, especially if we're moving to 2.80 as minimum version. Neither of those properties is present in the [RenderSettings API docs](https://docs.blender.org/api/blender2.8/bpy.types.RenderSettings.html#bpy.types.RenderSettings).

Reply:
Agreed